seedpod_ground_risk/core/plot_server.py

```diff
@@ -198,26 +198,45 @@ def compose_overlay_plot(self, x_range: Optional[Sequence[float]] = (-1.6, -1.2)
 
                     t0 = time()
                     self._progress_bar_callback(10)
+                    # TODO: This will give multiple data layers, these need to be able to fed into their relevent pathfinding layers
```

Comment:
Is this `TODO` complete? If not, would be good to complete a feature fully, otherwise we are just piling up technical debt that we have to clean up later anyway..